app/hooks/useClaimTo.tsx

```
@@ -0,0 +1,94 @@
+  const activeAccount = useActiveAccount();
+  const [error, setError] = useState<IClaimToError>();
+  const [isProcessing, setIsProcessing] = useState(false);
+
+  const handleClaim = async (args: IClaimToArgs) => {
+    const TRANSACTION_TIMEOUT = 30000;
+
+    (Object.keys(args) as (keyof IClaimToArgs)[]).forEach((key) => {
+      if (!args[key]) {
+        throw new Error(`${key} is required`);
+      }
+    });
```

Biome (1.9.4)

[error] Prefer for...of instead of forEach.

forEach may lead to performance issues when working with large arrays. When combined with functions like filter or map, this causes multiple iterations over the same type.

(lint/complexity/noForEach)